frontend/app/src/store/history/consts.ts

```diff
@@ -184,6 +184,10 @@ export const historyEventSubTypeData: ActionDataEntry[] = [
     label: i18n
       .t('transactions.events.history_event_subtype.return_wrapped')
       .toString()
+  },
+  {
+    identifier: HistoryEventSubType.REWARD,
```

Comment:
I don't understand here. Where is the rule that says RECEIVE + REWARD + 'votium' should use the given icon ?

Reply:
The counterparty icon goes here https://github.com/rotki/rotki/blob/a55efa5a8008ae069bded87fd21b7d7853b6df3e/frontend/app/src/store/history/consts.ts#L358

For the icon of the action look at https://github.com/rotki/rotki/blob/a55efa5a8008ae069bded87fd21b7d7853b6df3e/frontend/app/src/store/history/consts.ts#L279

Reply:
Thank you friend Yabir